puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman.groovy

```diff
@@ -33,14 +33,6 @@ def addGem() {
 
 def addSettings(settings) {
     sh "cp config/settings.yaml.example config/settings.yaml"
-
-    if (settings['locations']) {
-        sh "sed -i 's/:locations_enabled: false/:locations_enabled: true/' config/settings.yaml"
-    }
-
-    if (settings['organizations']) {
-        sh "sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/settings.yaml"
-    }
```

Comment:
The trouble is, this code is not isolated to nightly. That is, for example, this code is used on Katello branch PRs against older Foreman code. So we either leave this around for a few releases, or try to version lock it.

Reply:
The above is true for all places this is used.

Reply:
only we never actually use this file when running tests thanks to your commit from 3 years ago https://github.com/theforeman/foreman/pull/2528 :smile_cat:

Reply:
I think there may actually be on case where it does work, which is bats testing - i think in that case we run foreman with production environment rather than tests. That may also be the case for proxy tests but i'm not sure.

Reply:
The bats test shouldn't be using this groovy code at all. I am fine leaving it for now if we are not sure.